_posts/2013-11-22-sr2014-underway.md

```diff
@@ -11,7 +11,7 @@ The Student Robotics 2014 competition is now under way. This year we have
 three hundred students registered, across fifty four schools, and three
 different countries. They now have six months to design, prototype and build
 their robots, to take part in this years competition game. You can find more
-information about the people taking part on their [team pages](/teams).
```

Comment:
Note that this was a very different teams page -- a long time ago we had a page which listed all the teams that were taking part and there was a way for them to set status updates and the like.